Dfe.Academies.External.Web/Services/TempDataHelperService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text.Json;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+
+namespace Dfe.Academies.External.Web.Services;
+
+/// <summary>
+/// Helper service to put stuff in and out of TempData[]
+/// to handle serialization / de-serialization / type conversion /
+/// whether key found in TempData[]
+/// </summary>
+public class TempDataHelperService: ITempDataHelperService
```

Comment:
I think we should decide whether this is a helper or a service. I think it's a helper, in that all these methods could be static because there are no member fields or properties holding data. In which case, perhaps it could just be a static class rather than a singleton?